Marlin/Marlin_main.cpp

```diff
@@ -2834,7 +2834,7 @@ static void do_homing_move(const AxisEnum axis, float distance, float fr_mm_s=0.
 
   stepper.synchronize();
 
-  #if QUIET_PROBING
+  #if HAS_BED_PROBEQUIET_PROBING
```

Comment:
Hmm.  Wonder if it's better to just add a conditional so that QUIET_PROBING can't be set if HAS_BED_PROBE isn't?